lms/static/js/fixtures/accordion.html

```diff
@@ -1,11 +1,11 @@
 <div class="course-wrapper">
   <div class="accordion">
-    <a href="accordion-menu-1" role="button" class="button-chapter chapter is-open" aria-controls="accordion-menu-1">
+    <a href="accordion-menu-1" role="button" class="button-chapter chapter is-open" aria-controls="accordion-menu-1" aria-expanded="true">
```

Comment:
aria-expanded should be on the "button" control, not the content associated with it.